files/tasks/install-packit-deps.yaml

```diff
@@ -1,49 +0,0 @@
-# Copyright Contributors to the Packit project.
-# SPDX-License-Identifier: MIT
-
-# Install dependencies for packit
-- name: Download script setupcfg2rpm.py
-  ansible.builtin.get_url:
-    url: https://raw.githubusercontent.com/packit/deployment/main/scripts/setupcfg2rpm.py
```

Comment:
Is the `setupcfg2rpm.py` still used anywhere? If not, can it be deleted as well?

Reply:
I don't think it is still used, but there is a reference to it in [tokman containerfile](https://github.com/packit/tokman/blob/8db9af20f24f7e7eb855f25e8431f723c061f955/Containerfile#L6-L7), not sure why.

Reply:
I'm blind, it [is being used](https://github.com/packit/tokman/blob/8db9af20f24f7e7eb855f25e8431f723c061f955/Containerfile#L18) there. Should I keep it or try to remove it?

Reply:
Definitely keep.